src/installer.ts

```diff
@@ -114,7 +114,48 @@ async function downloadRelease(version: string): Promise<string> {
   return await tc.cacheDir(extPath, "protoc", version);
 }
 
-function getFileName(version: string): string {
+/**
+ *
+ * @param osArch - A string identifying the operating system platform for which the Node.js binary was compiled.
+ * See https://nodejs.org/api/os.html#osplatform for possible values.
```

Comment:
```suggestion
 * @param osArch - A string identifying operating system CPU architecture for which the Node.js binary was compiled.
 * See https://nodejs.org/api/os.html#osarch for possible values.
```

The description and reference URL were for the operating system platform/`os.platform()`, whereas this parameter is the architecture/`os.arch()`.